_data/signed/amon22.yaml

```diff
@@ -0,0 +1,2 @@
+name: amon22
+link: https://github.com/rms-support-letter/rms-support-letter.github.io
```

Comment:
I think it'd be better if you use your github profile as your link instead. You can still emphasize the letter by pinning the repository to your profile. :)